stdlib/public/runtime/SwiftObject.mm

```diff
@@ -414,37 +419,29 @@ - (id)performSelector:(SEL)aSelector withObject:(id)object1
                                                                  object2);
 }
 
-- (NSString *)description {
+- (id)description {
   return _getObjectDescription(self);
 }
-- (NSString *)debugDescription {
+- (id)debugDescription {
   return _getObjectDescription(self);
 }
 
-+ (NSString *)description {
++ (id)description {
   return _getClassDescription(self);
 }
-+ (NSString *)debugDescription {
++ (id)debugDescription {
   return _getClassDescription(self);
 }
 
-- (NSString *)_copyDescription {
+- (id)_copyDescription {
   // The NSObject version of this pushes an autoreleasepool in case -description
   // autoreleases, but we're OK with leaking things if we're at the top level
   // of the main thread with no autorelease pool.
   return [[self description] retain];
 }
 
 - (CFTypeID)_cfTypeID {
-  // Adopt the same CFTypeID as NSObject.
-  static CFTypeID result;
-  static dispatch_once_t predicate;
-  dispatch_once_f(&predicate, &result, [](void *resultAddr) {
-    id obj = [[NSObject alloc] init];
-    *(CFTypeID*)resultAddr = [obj _cfTypeID];
-    [obj release];
-  });
-  return result;
+  return (CFTypeID)1; //NSObject's CFTypeID is constant
```

Comment:
This is slightly unprincipled, but I think it's still better than the other way.

We should make sure that -_cfTypeID returns the right thing for our custom subclasses though. I'll double check that.

Reply:
Would it be worth adding an assert that it matches the value from CF, the way we do for `SWIFT_ISA_MASK`? https://github.com/apple/swift/blob/master/stdlib/public/runtime/SwiftObject.mm#L197